### PR TITLE
Seeds improvements

### PR DESCRIPTION
**Pull request by angelk**
Using real gpx files for sseds

### CODE REVIEW

**Comment by StefanMavrodiev** [comment]
Maybe we should have one "real" and one "dummy" seeder?

**Comment by angelk** [comment]
Reverted real tracks and implemented only "generated" ones

File: script/migration/seeds/TrackSeeder.php [hunk 1]
```diff
@@ -12,34 +13,101 @@ class TrackSeeder extends AbstractSeed
...
+                'last_check' => date('Y-m-d H:i:s', strtotime(sprintf("-%d hours", $i))),
+                'name' => 'Dummy track ' . $i,
                 'point_north_east_lat' => 0,
                 'point_north_east_lng' => 0,
                 'point_south_west_lat' => 0,
                 'point_south_west_lng' => 0,
-                'type' => $this::TYPE_CYCLING,
-                'created_at' => date('Y-m-d H:i:s', strtotime(sprintf("+%d hours", $i))),
-                'visibility' => $this::VISIBILITY_PUBLIC,
+                'type' => self::TYPE_CYCLING,
```
Comment (StefanMavrodiev) [inline script/migration/seeds/TrackSeeder.php]: Generate also hiking tracks?
`'type' => ($i % 2) ? self::TYPE_CYCLING : self::TYPE_HIKING`

Maybe increase count to 30?

Comment (angelk) [reply script/migration/seeds/TrackSeeder.php]: Added logic for TrackVisibility & TrackType

**Pull request merged**